### ISSUE TITLE
Mac build #8109 changes date created, date added, and date modified of Resume and Torrents files

### ISSUE THREAD

**Issue by gradu8ed** [transmission/transmission#3458]
Running MacOS 10.13.6
Been running Transmission 3.0 for years without any issues. Decided to try the new stable build #8109. Paused all open, completed and seeding transfers (approx 1.200) in Transmission 3.00. Downloaded and installed then opened build #6109, as noted there was  a delay in program startup, checked out the preferences and didn't see anything new and quit. Reopened Transmission 3.0 and went to unpause the torrents in process and instead of the 254 files in process there are thousands at which point I quit right away. Now;
library/application support/transmission/resume has 6,367 files and the creation date, date added, and date modified are today, 07/16/2022 between 3:43 PM to 4:13 PM
library/application support/transmission/torrents has 6,392 files and the creation date, date added, and date modified are today, 07/16/2022 between 4:00 PM to 4:08 PM, This file total seems reasonable since it was the results of torrents dating between now and 2015. 
My bad for not backing up the Transmission folder in application support. Any ideas on how to roll back/fix would be welcome.

**Comment by GaryElshaw** [comment]
Really afraid to ask (having once been there), no time machine backup?

**Comment by gradu8ed** [comment]
Nope. Just finishing up a NAS to set time machine up in the next few days. Too little too late I guess.

**Comment by GaryElshaw** [comment]
So sorry, it's a horrible feeling. Not on the scale you've experienced, but I recommend starting with a fresh directory and moving to the nightlies to future proof any moves to 4.0 you might want in the future. 

I've made the switch to using the nightlies and occasionally use 3.0. It seems like that particular bug only effects the first use, and subsequent moves back and forth are unaffected.
